packages/withoutHTML/src/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @description Cleanups a string from HTML tags.
+ * @param {String} text
+ * @example
+ * withoutDiacritics('<b>Test</b>') // returns 'Test'
```

Comment:
The documentation and the naming is wrong.

Reply:
Fixed it